tests/integration/partial_set_security_test.go

```diff
@@ -0,0 +1,176 @@
+package integration
+
+import (
+	"slices"
+	"testing"
+
+	"cosmossdk.io/math"
+	ccv "github.com/cosmos/interchain-security/v5/x/ccv/types"
+	"github.com/stretchr/testify/require"
+
+	icstestingutils "github.com/cosmos/interchain-security/v5/testutil/ibc_testing"
+
+	appConsumer "github.com/cosmos/interchain-security/v5/app/consumer"
+	appProvider "github.com/cosmos/interchain-security/v5/app/provider"
+)
+
+// we need a stake multiplier because tokens do not directly correspond to voting power
+// this is needed because 1000000 tokens = 1 voting power, so lower multipliers
+// will be verbose and harder to read because small token numbers
+// won't correspond to at least one voting power
+const stake_multiplier = 1000000
```

Comment:
Should we use camel case for `const`s? So, `stakeMultplier`?

Reply:
Sure